test/setup/README.md

```diff
@@ -0,0 +1,54 @@
+## Preparation
+***
+
+Prepare input for cortx_setup. There are two files in this folder.  
+a. cluster.yaml  
+b. config.yaml  
+
+Install cortx-utils and cortx-provisioner, cortx-components rpms.Make sure PYTHON_PATH includes src folder, if you are not using rpm.  
+```bash
+export CORTX_RELEASE_REPO=<<build-link of kubernetes branch>>
+yum install -y yum-utils
+yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
+yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
+cat <<EOF >/etc/pip.conf
+[global]
+timeout: 60
+index-url: $CORTX_RELEASE_REPO/python_deps/
+trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
+EOF
+yum install --nogpgcheck -y python3 cortx-prereq
+yum install --nogpgcheck -y cortx-py-utils python36-cortx-prvsnr
+yum install --nogpgcheck -y cortx-provisioner
+yum install --nogpgcheck -y cortx-motr cortx-hare cortx-s3server cortx-csm_agent cortx-csm_web
+```
+
+## Testing Manually
+
+Run apply command
+```bash
+cortx_setup.py config apply -f yaml://`pwd`/config.yaml -c yaml:///tmp/mp.conf
+cortx_setup.py config apply -f yaml://`pwd`/cluster.yaml -c yaml:///tmp/mp.conf
+```
+
+Run bootstrap command
+```bash
```

Comment:
Why do we need bash to execute python exec...

Reply:
done